haystack/nodes/connector/crawler.py

```diff
@@ -94,15 +95,15 @@ def __init__(
                 options.add_argument("--no-sandbox")
                 options.add_argument("--disable-dev-shm-usage")
                 self.driver = webdriver.Chrome(service=Service("chromedriver"), options=options)
-            except:
-                raise Exception(
+            except WebDriverException as exc:
+                raise NodeError(
```

Comment:
It looks like selenium raises it's own error type 😅 so I'm catching `WebDriverException` here.